include/xnnpack.h

```diff
@@ -1379,6 +1382,24 @@ enum xnn_status xnn_define_copysign(
   uint32_t output_id,
   uint32_t flags);
 
+
+/// Define a S32 Multiply node and add it to a Subgraph.
+///
+/// The Multiply node copies the first input and second input.
+///
+/// @param subgraph - a Subgraph object that will own the created Node.
+/// @param input1_id - Value ID for the first input tensor. The input tensor must be defined in the @a subgraph.
+/// @param input2_id - Value ID for the second input tensor. The input tensor must be defined in the @a subgraph.
+/// @param output_id - Value ID for the output tensor.
+/// @param flags - binary features of the Multiply node. No supported flags are currently defined.
+enum xnn_status xnn_define_vmultiply(
```

Comment:
This is not required. Please add to https://github.com/google/XNNPACK/blob/122235ee2ed02566ba50765262353eb12171e684/src/subgraph/multiply2.c#L249